sdk/servicebus/service-bus/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/service-bus",
   "sdk-type": "client",
   "author": "Microsoft Corporation",
-  "version": "7.4.1",
+  "version": "7.5.0",
```

Comment:
do we normally release a beta first?

Reply:
Since we havent discussed this with other languages yet, I would be ok with a beta. Otherwise, it is a straight forward feature that we could have gone GA directly with

Reply:
Changed to 7.5.0-beta.1 for now.